tests/unit/util/test_rule_dry_runner.py

```diff
@@ -0,0 +1,101 @@
+# pylint: disable=missing-docstring
+# pylint: disable=no-self-use
+import json
+import logging
+import os
+
+from logprep.util.rule_dry_runner import DryRunner
+
+
+class TestRunLogprep:
+    def test_dry_run_accepts_json_as_input(self, tmp_path, capsys):
+        config = "tests/testdata/config/config-dry-run.yml"
```

Comment:
could we get this config to this file to be aware what this config is without searching the config file?